frontend/packages/console-app/src/components/guided-tours/GuidedTourDrawer.scss

```diff
@@ -0,0 +1,11 @@
+.co-guided-tour-drawer {
+  &__title {
+    display: inline-block;
+    padding-right: var(--pf-global--spacer--md) !important;
+  }
+
+  &__subtitle {
+    display: inline-block;
+    color: #9c9c9c !important;
```

Comment:
Use a PF color variable. Consult UX or look for an equivalent in PF.

Reply:
done